### PR TITLE
Add initial dev conventions (for tests)

### PR DESCRIPTION
**Pull request by ravwojdyla**
Related: #75

### CODE REVIEW

**Comment by ravwojdyla** [comment]
@tomwhite I have a follow up to this PR in #110 which elaborates further on the `Development conventions`, would you still prefer to have this moved (I am fine with that just wanted to double check).

**Comment by tomwhite** [comment]
I think it can all go under `Contributing to code`.

**Comment by ravwojdyla** [comment]
@tomwhite okie

File: docs/contributing.rst [hunk 1]
```diff
@@ -146,6 +146,19 @@ are not covered by tests.
 
 The test suite is run automatically by CI.
 
+Test files live in ``tests`` directory, test filename naming convention:
```
Comment (jeromekelleher) [inline docs/contributing.rst:149]: They're in the ``sgkit/tests`` directory, right?

Comment (ravwojdyla) [reply docs/contributing.rst]: Fixed.

**Pull request merged**